docs/public-networks/how-to/troubleshoot/trace-transactions.md

```diff
@@ -0,0 +1,48 @@
+---
+description: How to trace transactions
+---
+
+# Trace transactions
+
+To get detailed information about transaction processing, use the
+[`TRACE` API](../../reference/api/index.md#trace-methods).
+Enable the `TRACE` API using the
+[`--rpc-http-api`](../../reference/cli/options.md#rpc-http-api) or
+[`--rpc-ws-api`](../../reference/cli/options.md#rpc-ws-api) command line options.
+
+The `TRACE` API has two sets of trace calls, [ad-hoc tracing APIs](#ad-hoc-tracing-apis) and
+[transaction-trace filtering APIs](#transaction-trace-filtering-apis).
+
+## Ad-hoc tracing APIs
+
+These APIs allow different diagnostic options when tracing calls or transactions.
```

Comment:
These APIs allow you to use the `trace`, `vmTrace`, or `stateDiff` diagnostic options when tracing calls or transactions.